src/components/inputs/Combobox/Combobox.test.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Combobox from './Combobox';
+import TestComponentPropUtils from '../../../utils/TestComponentPropUtils';
+
+describe('Combobox', () => {
+	// it('renders without crashing', () => {
```

Comment:
Does this commented out section need to be re-written? Or, can it be removed?

Reply:
Deleted this test file for now, will probably add some tests later this afternoon or tomorrow